js/scripts.js

```diff
@@ -0,0 +1,12 @@
+var list = document.getElementById('list');
+var add = document.getElementById('addElem');
+var element = document.createElement('click','li')[0];
+var length = list.getElementsByTagName('li');
+
+addElem.addEventListner('click', create(addElem));
```

Comment:
this function will be create immediately, you should pack it into an anonymous function